integrations/tensorflow/iree_tf_compiler/TFL/test/lower_global_tensors.mlir

```diff
@@ -1,7 +1,7 @@
 // RUN: iree-opt-tflite --split-input-file --allow-unregistered-dialect --pass-pipeline='iree-tflite-lower-global-tensors' %s | FileCheck %s
 
 module {
-  // CHECK: iree_input.global private mutable @__iree_flow_Variable = dense<1.000000e+00> : tensor<16x16xf32>
+  // CHECK: ml_program.global private mutable @__global_Variable(dense<1.000000e+00> : tensor<16x16xf32>)
```

Comment:
Doesn't the name here serve the same purpose as the VarHandle below?

Reply:
It serves the same purpose however VarHandles are variables, this allows them to be passed into functions or operation regions. E.g. you can execute a `tfl.if` operation that returns a varhandle, then write to that varhandle. In MLProgram we assume all reads/writes have the global tensor known at compile time.